consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/util/ResponsesSummaryUtil.java

```diff
@@ -0,0 +1,138 @@
+package uk.gov.moj.sdt.cmc.consumers.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import uk.gov.moj.sdt.cmc.consumers.converter.XmlToObjectConverter;
+import uk.gov.moj.sdt.cmc.consumers.model.claimdefences.ClaimDefencesResult;
+import uk.gov.moj.sdt.ws._2013.sdt.targetapp.submitqueryresponseschema.SubmitQueryResponseType;
+
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+@Component
+public class ResponsesSummaryUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ResponsesSummaryUtil.class);
+
+    private XmlToObjectConverter xmlToObjectConverter;
+
+    public ResponsesSummaryUtil(XmlToObjectConverter xmlToObjectConverter) {
+        this. xmlToObjectConverter =  xmlToObjectConverter;
+    }
+
+    public String getSummaryResults(Object mcolSubmitResponseTypeObject,
+                                    Object cmcSubmitResponseTypeObject) {
```

Comment:
Change signatures of this method to 
getSummaryResults(SubmitQueryResponseType mcolSubmitResponseTypeObject,
                                    List<ClaimDefencesResult> cmcSubmitResponseTypeObject)

Reply:
SubmitQueryResponseType not accessible so settled for SubmitQueryResponse. 
The signature is now getSummaryResults(SubmitQueryResponse mcolSubmitResponse,
List<ClaimDefencesResult> cmcResults)